config/OPM-Reconfig.cfg

```diff
@@ -1,6 +1,6 @@
 // Updated 2019/03/18 by Eridan/HSJasperism
 // Tested with OPM 2.2.2 & CA 1.6
-// OPM older than 2.2.2 of OPM will include dated configs for CA or Kopernicus Asteroids
+// OPM older than 2.2.2 will include dated configs for CA or Kopernicus Asteroids
 // CA older than 1.3 will fail to recognize spawn conditions
```

Comment:
CA older than 1.6 won't process the `#autoLOC_` tags, so that's a definite lower limit on the version.

Reply:
oh right, sorry about that